user_guide_src/source/outgoing/table.rst

```diff
@@ -71,6 +71,32 @@ to the Table constructor:
 
 .. literalinclude:: table/008.php
 
+Synchronizing row columns with heading
+======================================
+
+.. versionadded:: 4.4.0
+
+``setSyncRowsWithHeading(true)`` enables that each data value
+is placed in the same column as defined in ``setHeading()`` if an
+associative array was used as parameter. This is especially useful
+when dealing with data loaded via REST API where the order is not to
+your liking, or if the API returned too much data.
+
+If a data row contains a ``key`` which does not exist the heading,
+the value will be filtered out. Vise versa if a data row does not have key
+mentioned in heading, it places an empty cell for that spot.
```

Comment:
We use ` `` ` for source code or an item in source code. `key` is not in the source code.
If you really want to emphasize it, use `` `key` ``.

```suggestion
If a data row contains a key that is not present in the heading, its value is
filtered. Conversely, if a data row does not have a key listed in the heading,
an empty cell will be placed in its place.
```